js/Worksheet.js

```diff
@@ -1049,8 +1049,9 @@ class Worksheet extends HTMLElement {
     }
 
     toCSV() {
-        const data = this.sheet.dataFrame.getDataArrayForFrame(
-            this.sheet.dataFrame
+        const data = this.sheet.dataStore.getDataArray(
+            this.sheet.baseFrame.origin,
```

Comment:
Same as Excel method comment -- we need coordinate pairs, not Points

Reply:
see PR description